pages/treeherder.py

```diff
@@ -33,8 +34,20 @@ def job_result_status(self):
     def unclassified_failure_count(self):
         return int(self.selenium.find_element(*self._unclassified_failure_count_locator).text)
 
+    @property
+    def first_revision_date(self):
+        return self.selenium.find_element(*self._first_resultset_datestamp_locator).text
+
+    @property
+    def revision_date(self):
+        return self.selenium.find_element(*self._single_resultset_datestamp_locator).text
```

Comment:
I think we can just call `first_revision_date`, it just so happens that there's only one result, so the first result is still accurate.